akka-actor-tests/src/test/scala/akka/AkkaVersionSpec.scala

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class AkkaVersionSpec extends WordSpec with Matchers {
+
+  "The Akka version check" must {
+
+    "succeed if version is ok" in {
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.6")
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.7")
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.6.0")
+    }
+
+    "succeed if version is RC and ok" in {
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.6-RC1")
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.7-RC10")
+      AkkaVersion.require("AkkaVersionSpec", "2.4.19", "2.4.19-RC6")
+    }
+
+    "succeed if version is milestone and ok" in {
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.6-M1")
```

Comment:
should this succeed? someone is running with an old `2.5.6-M1` as current version but we have compiled with later `2.5.6`?

we should probably not support the other way either, we have compiled with old 2.5.6-M1 but later decide to remove a feature before the final 2.5.6 release

same with RCs

Reply:
Ah, true, I thought about it backwards.